eclass/python-utils-r1.eclass

```diff
@@ -1518,6 +1546,17 @@ epytest() {
 		fi
 	fi
 
+	# If we are using hypothesis (require use via EPYTEST_PLUGINS, since
+	# ebuilds may disable autoloading manually) *and* hypothesis-gentoo
+	# is available, use it to disable all health checks, to prevent the tests
+	# from failing randomly under load.
+	if has hypothesis "${EPYTEST_PLUGINS[@]}" &&
+		"${EPYTHON}" -c 'import hypothesis_gentoo' 2>/dev/null &&
```

Comment:
can't this be translated into `has_version`? Or maybe the usedep inside `[]` for the active EPYTHON makes it harder...

Reply:
Precisely that. I didn't really want to introduce logic for translating EPYTHON into USEDEPs for the sake of this.